### DISCUSSION TITLE
Setup with Nuxt

### DISCUSSION THREAD

**Discussion by paul-nallet** [chakra-ui/panda#852] [Q&A]
Hi !

i'm tying to setup panda with nuxt (v3). I'm following vue setup from documentation, it seems not working for now. 
have i missed something ?

**Comment by anubra266** [comment]
@paul-nallet what error exactly have you encountered

**Reply by paul-nallet** [reply]
i try with a new setup and it's orking well. i have missing something in the first project. 
thx
